package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./public/index.js",
   "scripts": {
     "start": "node src/server.js",
-    "dev": "NODE_ENV=development concurrently \" webpack-dev-server --progress --color \" \"nodemon --inspect src/server.js\"",
+    "dev": "NODE_ENV=development concurrently \" webpack-dev-server --color \" \"nodemon --inspect src/server.js\"",
```

Comment:
The `--progress` flag caused webpack to output considerable text to the console which obscures the comparatively tiny output from nodemon when there is an issue.

Reply:
I didn't realize that's why there was so much output! Good fix.

Reply:
I went looking for a cli argument to try to suppress some of that output and noticed that the --progress option we had was deliberately causing it, lol

Reply:
Should we either remove the "NODE_ENV=development" or actually add development/production flags in the .env?

Reply:
I'd argue that when running `npm run dev` we _want_ development mode, so I think that the current NODE_ENV is ok.

Reply:
Right, but we don't have a development or production environment specified. Should we just leave this as is until we create those?

Reply:
Ah, I see. Yeah, I don't think we need to worry about that at this point since we're not close to deploying.

Reply:
Also, I think webpack operates differently depending on the mode so we _do_ want to be passing it development currently. For example, it shows detailed error information on the frontend.